wp-content/themes/pub/wporg-learn-2024/src/sensei-meta-list/index.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Block Name: Sensei Meta List
+ * Description: Display the site meta data of a learn or course as a list.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\Theme\Learn_2024\Sensei_Meta_List;
+
+use Sensei_Utils;
+use Sensei_Reports_Overview_Service_Courses;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registers the block using the metadata loaded from the `block.json` file.
+ * Behind the scenes, it registers also all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/reference/functions/register_block_type/
+ */
+function init() {
+	register_block_type(
+		dirname( dirname( __DIR__ ) ) . '/build/sensei-meta-list',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render',
+		)
+	);
+}
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['postId'] ) ) {
+		return '';
+	}
+
+	$list_items = array();
+
+	if ( get_post_type() === 'course' ) {
```

Comment:
You can `postType` to block context for this, but not sure if it makes a difference

Reply:
I think they are the same value in this case, both fetching the current post type. Just one from the front end and the other from the DB. Thanks.
1126560